src/common/common.h

```diff
@@ -213,6 +218,11 @@ OQS_API int OQS_CPU_has_extension(OQS_CPU_EXT ext);
  */
 OQS_API void OQS_init(void);
 
+/**
+ * This function stops and frees OpenSSL threads resources in the correct order
```

Comment:
I [previously volunteered](https://github.com/open-quantum-safe/liboqs/pull/1942#issuecomment-2394057202) to work on the documentation for the new API—would you like me to push anything expanding on this this @ashman-p?

Reply:
Thanks @SWilson4, if there is additional verbage to add how about you address here as part of this PR? That way we it is done.

Reply:
> Thanks @SWilson4, if there is additional verbage to add how about you address here as part of this PR? That way it is done all at once.

I added a couple of sentences to the API documentation below.